core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -137,10 +137,14 @@ object ConsoleProducer {
       .withRequiredArg
       .describedAs("topic")
       .ofType(classOf[String])
-    val brokerListOpt = parser.accepts("broker-list", "REQUIRED: The broker list string in the form HOST1:PORT1,HOST2:PORT2.")
+    val brokerListOpt = parser.accepts("broker-list", "REQUIRED (only when using old producer): The broker list string in the form HOST1:PORT1,HOST2:PORT2.")
       .withRequiredArg
       .describedAs("broker-list")
       .ofType(classOf[String])
+    val bootstrapServerListOpt = parser.accepts("bootstrap-server", "REQUIRED (unless old producer is used): The bootstrap server list string in the form HOST1:PORT1,HOST2:PORT2.")
```

Comment:
I don't understand this. `bootstrap-server` and `broker-list` should be equivalent. The user should provide exactly one of the two. And it can be used with both old and new producers. `bootstrap-server` is recommended going forward for consistency with the console consumer. I think you did it right for the `VerifiableConsumer`, for example.

Also, we can call the val simply `bootstrapServerOpt`.